pkg/catalog/catalog.go

```diff
@@ -71,9 +71,31 @@ func (c *ContextCatalog) Upsert(plugin *cli.PluginInfo) error {
 		c.sharedCatalog.IndexByName[pluginNameTarget] = append(c.sharedCatalog.IndexByName[pluginNameTarget], plugin.InstallationPath)
 	}
 
+	// The "unknown" target was previously used in two scenarios:
+	// 1- to represent the global target (>= v0.28)
```

Comment:
With `v0.90` of runtime, we do not support plugins built using an `unknown` target.

```suggestion
	// 1- to represent the global target (>= v0.28 and < v0.90)
```